README.md

```diff
@@ -1,2 +1,8 @@
 # Test
-This a test for understand how create repositories and branches and mostly what do they do.
+
+Now, the first step is to create changes into the second branch for see what is it doing.
+
+
+I do this modification during the pull request
+
+I think that pull request just able people to do changes on your branch but may be I'm wrong I will see later.
```

Comment:
Are you getting to wonder if it's useful to create pull request ? Check https://help.github.com/articles/about-pull-requests/ for answers

Reply:
thx